common/src/main/java/com/wynntils/models/items/annotators/gui/ArchetypeAnnotator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.annotators.gui;
+
+import com.wynntils.core.text.StyledText;
+import com.wynntils.handlers.item.ItemAnnotation;
+import com.wynntils.handlers.item.ItemAnnotator;
+import com.wynntils.models.items.items.gui.ArchetypeItem;
+import com.wynntils.utils.mc.LoreUtils;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.item.ItemStack;
+
+public final class ArchetypeAnnotator implements ItemAnnotator {
+    // Test suite: https://regexr.com/7h12h
+    private static final Pattern ARCHETYPE_NAME = Pattern.compile("^§([a-r0-9])§l[A-Za-z ]+ Archetype$");
+    // Test suite: https://regexr.com/7h133
+    private static final Pattern ARCHETYPE_PATTERN = Pattern.compile("^§a✔ §7Unlocked Abilities: §f(\\d+)§7/1[56]$");
```

Comment:
Why not just specify `\\d` as the max? Or better yet, parse the max as well and return a CappedValue.